s3blobs/src/play/modules/s3blobs/S3Blob.java

```diff
@@ -158,6 +160,6 @@ public Object assemble(Serializable srlzbl, Object o) throws HibernateException
 
 	@Override
 	public Object replace(Object o, Object o1, Object o2) throws HibernateException {
-		throw new UnsupportedOperationException("Not supported yet.");
+		return o;
```

Comment:
We were getting this exception thrown when performing .merge on the parent object. According to the hibernate usertype java doc in most cases it's safe to return the original object here.